### DISCUSSION TITLE
gcc derleyicisinin OpenSuse sistemlere yüklenmesi

### DISCUSSION THREAD

**Discussion by Fersephone** [gusanmaz/BMB311_OS_2023#5] [Show and tell]
Konsola **gcc p1.c** yazdığınızda eğer dönüş olarak şunu alıyorsanız:

**If 'gcc' is not a typo you can use command-not-found to lookup the package that contains it, like this:
    cnf gcc**
 
**sudo zypper install gcc**

yaparak sorunu halledebilirsiniz.

**Comment by gusanmaz** [comment]
*OpenSuse dagitiminda
